symengine/functions.cpp

```diff
@@ -1181,7 +1182,7 @@ RCP<const Basic> trig_to_sqrt(const RCP<const Basic> &arg)
 }
 
 /* ---------------------------- */
-ASin::ASin(const RCP<const Basic> &arg) : TrigFunction(arg)
+ASin::ASin(const RCP<const Basic> &arg) : InverseTrigFunction(arg)
```

Comment:
A quick `grep` could found `TrigFunction`only at 
```
a/symengine/series_visitor.h
 public:
    void bvisit(const TrigFunction &f)
    {}
```
Is there any other place I need to fix?